README.md

```diff
@@ -140,6 +140,9 @@ Complete all of these tasks on your local home machine.
   * Rackspace Cloud
 
             sudo pip install pyrax
+  * Microsoft Azure
+
+            sudo pip install msrest msrestazure azure==2.0.0rc5
```

Comment:
I think it would be ok to just state `sudo pip install azure==2.0.0rc5` as the install requirement. `msrest`, and `msrestazure` are sub dependencies `azure==2.0.0rc5`. 

according to https://github.com/Azure/azure-sdk-for-python/issues/644 the need to explicitly install `msrest` etc, has been fixed as of Jan 2017.